packages/relay/tests/lib/relay.spec.ts

```diff
@@ -133,9 +120,11 @@ describe('RelayImpl', () => {
 
         expect(populatePreconfiguredSpendingPlansSpy.calledOnce).to.be.true;
         await expect(populatePreconfiguredSpendingPlansSpy.returnValues[0]).not.to.be.rejected;
+        const failedPreConfiguredSpendingPlansMessage = `Failed to load pre-configured spending plans: `;
+        const failedToLoadEnvVarSpendingPlan = `Failed to load HBAR_SPENDING_PLAN. JSON parse error: Unexpected token 's', "spendingPl"... is not valid JSON; `;
```

Comment:
Why have we changed the expected message here? 

We shouldn't be getting this error, instead we should expect this message:
```typescript
const cause = `Failed to parse JSON from ${path}: Unexpected token 'i', "invalid JSON" is not valid JSON`;
```

Reply:
Done.